### PR TITLE
Option destroyAlllayers not classified as a cache utility

### PR DESCRIPTION
**Pull request by SueChaplain**
- Add (cache utility) to option
- Slight rewording of cache utility intro and fixing of references to VM acronym.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: docs/xshareclasses.md [hunk 1]
```diff
@@ -26,11 +26,11 @@
...
 
-This option can take a number of parameters, some of which are cache utilities. Cache utilities perform the required operation on the specified cache, without starting the VM. You can combine multiple suboptions, which are separated by commas, but the cache utilities are mutually exclusive.
+This option can take a number of parameters, some of which are *cache utilities*. Cache utilities carry out specific operations on a specified cache without starting the Java virtual machine (VM). Although you can combine multiple suboptions, which are separated by commas, the cache utilities are mutually exclusive.
 
-When you are running cache utilities, the message `Could not create the Java virtual machine` is expected. Cache utilities do not create the virtual machine.
+When you run a cache utility, the message `Could not create the Java virtual machine` is expected because the utilities do not create the VM.
```
Comment (pshipton) [inline docs/xshareclasses.md:31]: This doesn't happen any more, I think it happened in early OpenJ9 builds.

Comment (SueChaplain) [reply docs/xshareclasses.md:31]: I think I got this message when I did a basic `java -Xshareclasses:help`. I had wanted to see if the utilities were marked in the help output per option. I used a recent build from Adopt too.

Comment (pshipton) [reply docs/xshareclasses.md:31]: What JVM did you use? I don't see it on recent JVMs running on xlinux.

Comment (SueChaplain) [reply docs/xshareclasses.md]: My mistake @pshipton - the message doesn't get printed. I had seen it when my macOS picked up system Java and got that response from HotSpot, not OpenJ9.

I've modified the PR. Please merge if you are happy with the changes.

**Review by pshipton — APPROVED** [review]

**Pull request merged**